save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DiktatDemoTool.kt

```diff
@@ -2,16 +2,19 @@ package com.saveourtool.save.demo.diktat
 
 /**
  * Enum that represents possible tool names in diktat-demo
+ * @property owner
+ * @property toolName
+ * @property version
  */
-enum class DiktatDemoTool {
+enum class DiktatDemoTool(val owner: String, val toolName: String, val version: String) {
```

Comment:
This is not actually a version &mdash; this is essentially a Git tag (which, generally, is different from the release name.

Could you please rename it accordingly?

```suggestion
enum class DiktatDemoTool(val owner: String, val toolName: String, val vcsTagName: String) {
```